docs/semgrep-ci.md

```diff
@@ -0,0 +1,60 @@
+# Semgrep CI
+
+[Semgrep CI](https://github.com/returntocorp/semgrep-action) is a wrapper around [Semgrep](https://github.com/returntocorp/semgrep) for running as a GitHub Action, in Gitlab, and in other CI providers and interfacing with [https://semgrep.dev](https://semgrep.dev). See [CI Providers](providers.md) to see how to setup Semgrep CI in the provider you already use.
+
+Semgrep CI adds a layer of git-awareness on top of Semgrep. It reviews only the changed files in pull requests with Semgrep whenever a new commit is added to them, and reports only issues that are newly introduced in that pull request.
+
+## Technical details
+
+Semgrep CI scans files in the current directory with [semgrep](https://github.com/returntocorp/semgrep), and exits with a return code of 1 if blocking issues are found.
+
+Findings are blocking by default. They can be set to non-blocking by changing the action in semgrep.dev/manage/policy.
+
+Semgrep-action has the option to report only new issues, added since a specific commit.
```

Comment:
I am slightly confused here (and in a few places below) by Semgrep-action vs. Semgrep CI.

Reply:
What if I remove this paragraph and below the paragraph below have something like 
```
When running in Github or GitLab, Semgrep CI is able to determine the base commit using environment variables set but the CI provider. For other providers, Semgrep CI will need to be explicitly told the base commit. See setup instructions for other providers [Link here] for more info.
```

Reply:
Thanks @brendongo, I think that’s helpful.

Reply:
Made this change